Changed on-click behaviors + cleared out unused code.

kailbal opened the pull request:
Instead of opening whenever the path was changed from the default, the Game button shows up whenever 'Games' is clicked. Don't have anything specific, but figure it'd be a good start once we actually know what we want on the navigation bar.

kailbal committed:
Changed on-click behaviors + cleared out unused code.

Instead of opening whenever the path was changed from the default, the Game button shows up whenever 'Games' is clicked. Don't have anything specific, but figure it'd be a good start once we actually know what we want on the navigation bar.

lynnbridge commented on frontend/src/components/NavMenu.vue:
```diff
@@ -11,10 +11,10 @@
       <v-list-item to="/about">About</v-list-item>
-      <v-list-item v-if="appBarDisplay" :to="`/info/${defaultGame}`">Game</v-list-item>
+      <v-list-item @click="gameSelected = !gameSelected"> Games </v-list-item>
+      <v-list-item v-if="gameSelected" :to="`/info/${defaultGame}`">Game</v-list-item>
```
Add a v-divider before the game to add some separation between the games selected. Make sure it also only shows up with the gameSelected check.

kailbal committed:
Update NavMenu.vue

kailbal committed:
Updated NavMenu

Cleared out old experimental stuff, now just have the 3 buttons as outlined in the plan (and added comments dictating things to do in the future)

kailbal commented:
reduced Navigation Bar assets to what is in the mock up, so now it's just the three buttons.

lynnbridge commented on frontend/src/components/NavMenu.vue:
```diff
@@ -8,13 +8,15 @@
       <v-list-item to="/about">About</v-list-item>
-      <v-list-item v-if="appBarDisplay" :to="`/info/${defaultGame}`">Game</v-list-item>
+      <!-- Game List: add route that goes to all games page-->
+      <v-list-item> Game List </v-list-item>
```
add `to="/games"

kailbal committed:
added /games

lynnbridge reviewed (approved):
Thanks for making those changes.
